### PR TITLE
Integrate new popup menu

### LINKED ISSUE #1775: Integrate new poupMenu
__What should we do?__

Integrate the new popupMenu in replace and align/distribute menus.

__Why should we do it?__

To improve look and feel of popup menu and prepare for replace anything.

### PR DESCRIPTION
**Pull request by smbea**
Closes #1775 

~~This is targeting `diagram-js@develop` since [this bug fix](https://github.com/bpmn-io/diagram-js/issues/695) was added after the alpha release. My idea is to pre-approve this to make sure the new popup menu is working as expected so we can have the stable release of `diagram-js`, and later integrate it here.~~

Integrate ~~`diagram-js@11.0.0`~~ `diagram-js@11.1.0`

Following the title discussion in the issue, I decided not to add the align title for now.

### CODE REVIEW

File: lib/features/context-pad/ContextPadProvider.js [hunk 1]
```diff
@@ -396,7 +396,10 @@ ContextPadProvider.prototype.getContextPadEntries = function(element) {
               cursor: { x: event.x, y: event.y }
             });
 
-            popupMenu.open(element, 'bpmn-replace', position);
+            popupMenu.open(element, 'bpmn-replace', position, {
+              title: 'Change element',
```
Comment (nikku) [inline lib/features/context-pad/ContextPadProvider.js:400]: Should be translated.

File: lib/features/align-elements/AlignElementsContextPadProvider.js [hunk 2]
```diff
@@ -59,7 +59,9 @@ AlignElementsContextPadProvider.prototype._getEntries = function(elements) {
             }
           });
 
-          self._popupMenu.open(elements, 'align-elements', position);
+          self._popupMenu.open(elements, 'align-elements', position, {
```
[the remark this answers is not in the document]
Comment (nikku) [reply lib/features/align-elements/AlignElementsContextPadProvider.js:62]: Merged.

**Comment by barmac** [comment]
I am looking into this right now.

**Comment by barmac** [comment]
https://user-images.githubusercontent.com/28307541/203007560-46020f63-1b4d-47fe-9dd0-7dae0cdc21ef.mov

Arrow keys don't work in the menu. Is this expected?

**Comment by barmac** [comment]
https://user-images.githubusercontent.com/28307541/203008059-0a87fbaf-dfe2-47b3-a841-885bc1c347ae.mov

Second click on the input closes popup menu though it shouldn't.

**Comment by nikku** [comment]
What @barmac reported earlier (mouse vanished, still no keyboard selection) looks like a bug, not a browser quirk :see_no_evil:. Fixed behavior showcased here:

capture Y1J0hd_optimized

**Comment by nikku** [comment]
Some work in progress fixing issues is pushed at https://github.com/bpmn-io/diagram-js/pull/706.

**Comment by nikku** [comment]
Adjusted topic on these commits.

**Review by nikku — COMMENTED** [review]
User testing this I found the positioning of the replace menu to be off (random place from what it looks like), cf. screens below.

Not sure if this is something we want to adjust.


#### Reference

capture 5cKu8K_optimized


#### Actual (this PR)

capture uWMYEX_optimized

**Review by barmac — APPROVED** [review]
LGTM

https://user-images.githubusercontent.com/28307541/203785680-42595d14-6a4f-4532-8880-3b63c8d3f414.mov

**Review by nikku — APPROVED** [review]
Looks good to me, too :medal_sports:

**Pull request merged**